src/rcheevos/operand.c

```diff
@@ -160,7 +209,7 @@ int rc_parse_operand(rc_operand_t* self, const char** memaddr, uint8_t is_indire
       }
       allow_decimal = 1;
       /* fall through */
-    case 'v': case 'V': /* signed integer constant */
+    case 'v': case 'V': /* signed integer constant or variable */
```

Comment:
How does this support `variable`?

Reply:
Stale comment from noodling around before settling on { } and I missed it during my comment cleanup passes.